hyp3lib/get_dem.py

```diff
@@ -260,19 +177,8 @@ def get_dem(x_min, y_min, x_max, y_max, outfile, post=None, processes=1, dem_nam
     p.close()
     p.join()
 
-    # os.system("gdalbuildvrt temp.vrt DEM/*.tif")
-    if "SRTMGL" in demname:
-        nodata = -32768
-    elif "GIMP" in demname:
-        nodata = None
-    elif "REMA" in demname:
-        nodata = 0
-    elif "NED" in demname or "EU_DEM_V11" in demname:
-        nodata = -3.4028234663852886e+38
-    else:
-        raise DemError(f'Unable to determine NoData value for DEM {demname}')
-
-    write_vrt(demproj, nodata, tile_list, poly_list, 'temp.vrt')
+    tiles = [f'DEM/{tile}.tif' for tile in tile_list]
+    subprocess.check_call(['gdalbuildvrt', 'temp.vrt'] + tiles)
```

Comment:
Are you still setting the nodata value for the different DEMs?

Reply:
`gdalbuildvrt` reads the nodata value out of the individual tiles and sets it in the resulting VRT; so far I've checked SRTMGL1 and NED13 and they still give the same results.  I intend to run additional tests after merging to develop/